src/control/cmd/daos_server/storage_scm.go

```diff
@@ -113,13 +126,15 @@ func (cmd *prepareSCMCmd) preparePMem(prepareBackend scmPrepareResetFn) error {
 }
 
 func (cmd *prepareSCMCmd) Execute(_ []string) error {
-	if err := cmd.init(cmd.MustLogCtx()); err != nil {
-		return err
+	if cmd.ctlSvc == nil {
+		// If specific storage control service is not supplied with command, do init().
+		if err := cmd.init(cmd.MustLogCtx()); err != nil {
+			return err
+		}
```

Comment:
suggestion - Would it make sense to do this nil check inside the `cmd.init()` function, since we seem to be doing it nearly everywhere?

Reply:
done